packages/recurring-payments-job/src/recurring-payments-processor.js

```diff
@@ -89,3 +99,16 @@ const preparePayment = (agreementId, transaction) => {
 
   return result
 }
+
+const processRecurringPaymentStatus = async record => {
+  const agreementId = record.entity.agreementId
+  const paymentId = getPaymentId(agreementId)
+  const status = await getPaymentStatus(paymentId)
+  console.log(`Payment status for ${paymentId}: ${JSON.stringify(status.state.status)}`)
```

Comment:
`status.state.status` is somewhat impenetrable - if rather than doing `const status = ` on the previous line, you do `const { status: { state: { status } } } = ` you'll have just the one status variable